sdk/nodejs/tsconfig.json

```diff
@@ -99,6 +99,7 @@
         "tests/runtime/props.spec.ts",
         "tests/runtime/settings.spec.ts",
         "tests/runtime/langhost/run.spec.ts",
+        "tests/runtime/langhost/cases/069.ambiguous_entrypoints/index.ts",
```

Comment:
The other cases aren't here? Are they all .js files?

Reply:
They are ALL .js files.

Reply:
(I also wish I didn't need to add this file here)